en/docs/get-started/configure-sms-otp-authenticator.md

```diff
@@ -0,0 +1,85 @@
+This document provides step by step instructions to setup SMS OTP configurations
+
+## Sign in to the Identity Server
+
+1. Sign in to the Management Console of WSO2 Identity Server at [https://localhost:9446/carbon](https://localhost:9446/carbon)
+
+2. Use the default super admin credentials as follows:
+    - Username: admin@wso2.com
+    - Password: wso2123
+
+    !!!note
+        The above login credentials are for testing purposes only. It is recommended to change the login credentials in
+        a production environment.
+
+### Step 1: Adding SMS OTP Identity Provider
+
+1. Go to the **Main menu** -> **Identity** -> **Identity Providers** -> **Add**.
+
+      ![add_identity_providers](../assets/img/get-started/quick-start-guide/go-to-add-identity-providers.png)
+
+2. Fill the **Basic Information** section and name this identity provider as **SMSAuthentication**
+
+      ![sms_authentication_idp](../assets/img/get-started/quick-start-guide/sms-authentication-idp.png)
+
+3. Expand the **Federated Authenticators** -> **SMS OTP Configuration** section
```

Comment:
Let's add full stop where we have used complete senteces. 
Pls check all the places in this doc and its duplicate